### DISCUSSION TITLE
i need to clone repo to my local

### DISCUSSION THREAD

**Discussion by bhanuraina** [DevSkillsHQ/discusssions#32] [Help] [answered]
I want to clone repo https://github.com/ffc1e12/transaction-management-backend-dev-ops-updated-instructions-6g2i3-ah13g4  to my local 

please suggest.

**Comment by fenske** [comment, marked as answer]
Hi there. 👋 Here's how you do it: https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository.

**Comment by bhanuraina** [comment]
Hi ,

the issue is as below :

***@***.****:*~*$ git clone
https://github.com/ffc1e12/transaction-management-backend-dev-ops-updated-instructions-6g2i3-ah13g4

Cloning into
'transaction-management-backend-dev-ops-updated-instructions-6g2i3-ah13g4'...

Username for 'https://github.com': bhanuraina

Password for ***@***.***':

remote: Write access to repository not granted.

fatal: unable to access '
https://github.com/ffc1e12/transaction-management-backend-dev-ops-updated-instructions-6g2i3-ah13g4/':
The requested URL returned error: 403


please help.

On Thu, Aug 10, 2023 at 1:06 PM Anton Fenske ***@***.***>
wrote:

> Hi there. 👋 Here's how you do it:
> https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository
> .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/DevSkillsHQ/discussions/32#discussioncomment-6690228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGIVSHE6XZJ4FVWLF3BFPODXUS6CDANCNFSM6AAAAAA3LJT5YA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards,

Bhanu Pratap Raina
Software Engineer

**Reply by fenske** [reply]
I'd recommend cloning your repo via ssh. Here's how to do it: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Basically, you'll need to install an ssh key locally (unless you already have one). And then, import it to your GitHub account as per the guide above. Then you'll be able to clone your repository with:

```
git clone git@github.com:ffc1e12/transaction-management-backend-dev-ops-updated-instructions-6g2i3-ah13g4.git
```

Alternatively, you can work in an in-browser IDE via Gitpod. Here's how: [link](https://github.com/ffc1e12/transaction-management-backend-dev-ops-updated-instructions-6g2i3-ah13g4/#gitpod-environment).
